### PR TITLE
8.0 azure theme text & link contrast fix - light theme only

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
## Previous Behavior

Low vision people and people sensitive to colors will find it difficult to recognize the control and differentiate between the surrounding text and link if the luminosity ratio is less than 3:1. Solution is to change body text in light theme only to #292827 to bring contrast ratio to 3:1. 

Encountered high contrast dark theme visual bug for link while working on this PR: on hover, link turns black, which blends in with background color. 
Link rest
image

Link hovered (turns black)
image

## New Behavior

1.) Updated the primary text color (light theme only) to #292827 to bring contrast ratio to 3.0 when used with inline <Link> in order to bring parity to bug found in KO controls. We are not updating text universally (for example, <Label/> control) because KO will not be doing it across their controls either. 
image

2.) Updated the color of links in message bar (Infobox in KO) <Link> colors updated for light theme only, maintained in other themes.

Rest link in message bar
image

Hover link in message bar
image

3.) To fix HCD link color, I aligned link hover to that of the portal, which is the same as rest (#) 
Link rest
image

Link hovered (no color change)
image

Helpful table:
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/c88ed5c283672106e237177f59a8bf356d6d9fc6'>c88ed5c283672106e237177f59a8bf356d6d9fc6</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Review by robtaft-ms — APPROVED** [review]
Great work thank you!

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by Jacqueline-ms** [comment]
Hi @TristanWatanabe - we have a high pri A11Y bug that this PR remedies. Is there any way we can prioritize approving this Typically, we had someone from my team approve these PRs and then I was able to merge in myself, but it looks like we now need additional reviews. Thanks!

File: packages/web-components/docs/api-report.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ export const ambientShadow = "0 0 2px rgba(0, 0, 0, 0.14)";
 // @internal
 export class Anchor extends Anchor_2 {
     // @public
-    appearance: AnchorAppearance;
+    appearance?: AnchorAppearance;
```
Comment (TristanWatanabe) [inline packages/web-components/docs/api-report.md:244]: Curious as to what triggered this change? It seems like the overall changes in the PR shouldn't have affected the `web-components` package API in any way. This is also why the web-components folk's approval has been required for this PR to get merged in

Comment (Jacqueline-ms) [reply packages/web-components/docs/api-report.md:244]: Thank you for getting back to me. Good question, I am unfamiliar with this package. I am going to cherry pick some of these changes into a fresh branch and see if this happens again.

**Comment by Jacqueline-ms** [comment]
Closing PR and re opening here https://github.com/microsoft/fluentui/pull/27947

**Pull request closed without merging**